test/stdlib/tgmath.swift.gyb

```diff
@@ -248,10 +248,26 @@ MathTests.test("${T}") {
 % for T in ['Float', 'Float80']:
 MathTests.test("gradient_${T}") {
   expectEqualWithTolerance(7.3890560989306502274, gradient(at: 2.0 as ${T}, in: exp), ulps:16)
+  expectEqualWithTolerance(2.772588722239781145, gradient(at: 2.0 as ${T}, in: exp2), ulps:16)
```

Comment:
```suggestion
  expectEqualWithTolerance(2.772588722239781145, gradient(at: 2.0 as ${T}, in: exp2), ulps: 16)
```
Same for all other occurrences.

Reply:
fixed on c92e34b5501ed136583f1e6e24f031f2dff53161